### DISCUSSION TITLE
[Newbie] Says tsserver is installed but doesn't seem to work on .ts files

### DISCUSSION THREAD

**Discussion by nsirimo** [williamboman/nvim-lsp-installer#766] [Q&A]
### Spec
M1 MBP on Monterey 12.4
Using NVChad (which uses nvim-lsp-installer).
Screenshots attached.

I'd like to say sorry if this is a dumb question but I'm not sure what to search and I've looked at all the closed issues already with no luck.

### Problem
I used the interactive installer :LspInstallInfo and it shows that my tsserver is installed. I tried restarting my nvim but it doesn't seem to be working for my .ts files. When I use LspInfo, it doesn't show any activated lsps.

![Screen Shot 2022-06-09 at 1 30 26 PM](https://user-images.githubusercontent.com/24559644/172939711-158c79bc-a22b-4e70-932d-49b82d94136b.png)
![Screen Shot 2022-06-09 at 1 30 10 PM](https://user-images.githubusercontent.com/24559644/172939717-8ef39708-45bb-49f8-9aab-81acc58bb8be.png)

**Comment by danpf** [comment]
impossible to say without more info, but try looking at the bottom of your listed "language client log" file for more info.

my guess is that you haven't fully set up nvchad. On its website follow the info on the lsp stuff page, and check out the maintainers dotfiles repo for more ideas of what could be wrong.

**Comment by williamboman** [comment]
Hello! You'll have to set up the servers as well. It seems like you've only set up `sumneko_lua`. This is most easily done via https://github.com/neovim/nvim-lspconfig which hosts 1) a collection of default configurations for servers, and 2) logic as to when and how to start servers. See this quickstart in lspconfig as an example: https://github.com/neovim/nvim-lspconfig#quickstart.
